reactor-core/src/test/java/reactor/core/publisher/TailCallSubscribeTest.java

```diff
@@ -197,7 +197,7 @@ public void repeat() throws Exception {
 				.startsWith(
 						tuple(Thread.class.getName(), "getStackTrace"),
 						tuple(stackCapturingPublisher.getClass().getName(), "subscribe"),
-						tuple(Flux.class.getName(), "subscribe"),
+						tuple(InternalFluxOperator.class.getName(), "subscribe"),
```

Comment:
unrelated change? maybe add a `[polish]` commit and tag as `for/merge-with-rebase`?

Reply:
related, actually... it does change the stacktrace because now it uses `subscribe(CoreSubscriber)`